packages/jss-plugin-syntax-global/src/index.js

```diff
@@ -1,12 +1,24 @@
-import {RuleList} from 'jss'
+// @flow
+import {
+  RuleList,
+  type Plugin,
+  type RuleOptions,
+  type ParentRule,
+  type StyleRule,
+  type BaseRule
+} from 'jss'
 
 const propKey = '@global'
 const prefixKey = '@global '
 
-class GlobalContainerRule {
+class GlobalContainerRule implements ParentRule {
```

Comment:
Mb instead of ParentRule call it ContainerRule?

Reply:
yeah that's better 👍